digitransit-search-util/packages/digitransit-search-util-execute-search-immidiate/index.js

```diff
@@ -404,7 +407,9 @@ export function getSearchResults(
     }
   }
 
-  if (allTargets || targets.includes('Stops')) {
+  if (allTargets || targets.includes('Stops') || targets.includes('Stations')) {
+    const useStops = targets.includes('Stops');
+    const useStations = targets.includes('Stations');
```

Comment:
```suggestion
  const useStops = targets.includes('Stops');
  const useStations = targets.includes('Stations');
  if (allTargets || useStops  || useStations ) {
```
I think this would improve the readability of the if-clause and reduce the congnitive complexity a bit.